src/trustme/__init__.py

```diff
@@ -229,7 +229,7 @@ class CA:
 
     def __init__(
         self,
-        parent_cert: Optional["CA"] = None,
```

Comment:
Sphinx seems to have some trouble with the stringy annotations (here and below). Since the file uses `from __future__ import annotations` I figure it's OK to remove.